.vscode/settings.json

```diff
@@ -6,6 +6,11 @@
   "editor.wordWrapColumn": 110,
   "files.trimTrailingWhitespace": true,
   "files.insertFinalNewline": true,
+  "[typescript]": {
+    "editor.codeActionsOnSave": {
+      "source.organizeImports": "never"
```

Comment:
I noticed you don't "organize imports" in this project.
So I added this setting to override my default setting.
It might be just a "me" problem, but I figured it makes it makes the way of working more explicit in this project and avoids having to comment on PRs: "Please put back the imports the way they were before"